samples/excel/89-preview-apis/chart-formatting.yaml

```diff
@@ -0,0 +1,328 @@
+                }
+
+                chart.dataLabels.position = "Center";
+                chart.dataLabels.separator = "\n";
+                chart.dataLabels.format.font.color = "#D9D9D9";
+            });
+        }
+
+        async function changeColor() {
+            await Excel.run(async (context) => {
+
+                var worksheet = context.workbook.worksheets.getActiveWorksheet();
+                var chart = worksheet.charts.getItem("SlopeChart");
+                var increase = worksheet.getRange("I6:I15");
+                var increaseRate = worksheet.getRange("J6:J15");
+                increase.load("values");
+                increaseRate.load("values");
+                chart.series.load("count");
+                await context.sync();
+
+                // find the countries with the highest GDP and the fasting GDP growth
+                var max = increase.values[0][0];
+                var maxIndex = 0;
+                var fast = increaseRate.values[0][0];
+                var fastIndex = 0;
+                
+                for (var i = 1; i < 10; i++){
+                    if (increase.values[i][0] > max) {
+                        max = increase.values[i][0];
+                        maxIndex = i;
+                    }
+                    if (increaseRate.values[i][0] > fast) {
+                        fast = increaseRate.values[i][0];
+                        fastIndex = i;
+                    }
+                }
+
+                // color everything grey before highlighting results
+                for (var i = 0; i < chart.series.count; i++){
+                    chart.series.getItemAt(i).format.line.color = "#636363";
+                    chart.series.getItemAt(i).dataLabels.format.font.color = "#636363";
+                }
+
+                var maxSeries = chart.series.getItemAt(maxIndex);
+                maxSeries.format.line.color = "#4472C4";
+                maxSeries.dataLabels.format.font.color = "#D9D9D9";
+                var fastSeries = chart.series.getItemAt(fastIndex);
+                fastSeries.format.line.color = "#75AB3B";
+                fastSeries.dataLabels.format.font.color = "#D9D9D9";
+
+                maxSeries.load();
+                fastSeries.load();
+                await context.sync();
+                maxSeries.name += " (Greatest Growth)";
+                fastSeries.name += " (Greatest Relative Growth)";
+            });
+        }
+
+        async function clearChart() {
+            await Excel.run(async (context) => {
+                var worksheet = context.workbook.worksheets.getActiveWorksheet();
+                var charts = worksheet.charts;
+                charts.load("count");
+                await context.sync();
+                
+                for (var i = 0; i < charts.count; i++) {
+                    charts.getItemAt(0).delete();
+                }
+            });
+        }
+
+        async function addSampleData() {
+            try {
```

Comment:
I don't think the try/catch _inside_ this method is needed because the whole `addSampleData `is passed to the `tryCatch()` method.